### PR TITLE
connect backend to frontend

### PR DESCRIPTION
**Pull request by KantorSerhiy**

### CODE REVIEW

**Comment by KantorSerhiy** [comment]
cannot add scrin to pr(
but i can show it in slack;)

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: backend/cinema_service/settings.py [hunk 1]
```diff
@@ -175,3 +179,8 @@
     "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
     "ROTATE_REFRESH_TOKENS": False,
 }
+
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:5173",
+    "http://127.0.0.1:5173",
```
[the remark this answers is not in the document]
Comment (KantorSerhiy) [reply backend/cinema_service/settings.py:185]: sry, forget delete it.
Had problems with upload image, tried all)

File: backend/Dockerfile [hunk 2]
```diff
@@ -23,6 +23,6 @@ RUN adduser \
     django-user
 
 RUN chown -R django-user:django-user /vol/
-RUN chmod -R 755 /vol/web/
+RUN chmod -R 777 /vol/web/media/
 
 USER django-user
```
Comment (ArsenPidhoretskyi) [inline backend/Dockerfile:28]: The same new line)

**Review by ArsenPidhoretskyi — APPROVED** [review]
Good job)

**Pull request closed without merging**